### DISCUSSION TITLE
How to left-align text in buttons

### DISCUSSION THREAD

**Discussion by thr3a** [mantinedev/mantine#2177] [Q&A] [answered]
Hello
By default the text in the buttons is centered, is there a way to make it left-aligned?

```html
<div style={{ width: 200 }}>
  <Button
    fullWidth
    color="orange"
    leftIcon={<IconEdit></IconEdit>}
  >
    Blog
  </Button>
</div>
```

I have tried creating text inside the button and setting align=left but the text is not left aligned. (`<Text align="left">Blog</Text>`)

Current
<img width="234" alt="スクリーンショット 2022-08-20 13 16 43" src="https://user-images.githubusercontent.com/6120301/185728953-92ac3ff2-6da7-4a8e-93cd-0a357abcae23.png">

Expected Results
![スクリーンショット 2022-08-20 13 16 43のコピー](https://user-images.githubusercontent.com/6120301/185728959-b1778d9b-9278-4247-85e1-24630aab8912.png)

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/stupefied-lucy-jp5d4i?file=/src/App.tsx

**Reply by thr3a** [reply]
Thanks for the sample code!

I understood that I can adjust the margin between the icon and the text in the button.

But I want the icon and text in the button to be left-aligned.
I checked the component props of button but could not find anything like that.

**Reply by rtivital** [reply]
https://codesandbox.io/s/old-glade-orezut?file=/src/App.tsx

**Reply by thr3a** [reply]
Thanks!!
